pkg/invariants/network/disruptionpodnetwork/invariant.go

```diff
@@ -90,13 +92,9 @@ func (pna *podNetworkAvalibility) StartCollection(ctx context.Context, adminREST
 	}
 	numNodes := int32(len(nodes.Items))
 
-	// disable the poller until can get the image sorted out.
-	// TODO re-enable
-	if false {
-		podNetworkPollerDeployment.Spec.Replicas = &numNodes
-		if _, err = pna.kubeClient.AppsV1().Deployments(pna.namespaceName).Create(context.Background(), podNetworkPollerDeployment, metav1.CreateOptions{}); err != nil {
-			return err
-		}
+	podNetworkPollerDeployment.Spec.Replicas = &numNodes
+	if _, err = pna.kubeClient.AppsV1().Deployments(pna.namespaceName).Create(context.Background(), podNetworkPollerDeployment, metav1.CreateOptions{}); err != nil {
+		return err
```

Comment:
How is this going to behave on single-node/microshift etc? I'm assuming we'll just get the hairpin results as there's nothing else to talk to?